velox/dwio/parquet/tests/writer/SinkTests.cpp

```diff
@@ -16,9 +16,8 @@
 
 #include <utility>
 
+#include "ParquetWriterTestBase.h"
```

Comment:
Use full paths: velox/...

Reply:
Fixed, sorry that I did not notice here.